File: Calculate_Money_in_Leetcode_Bank.java
class Solution {
        public int totalMoney(int n) {
        int weeks = n / 7;
        int days_left = n % 7;
        return (28 * weeks + 7 * weeks * (weeks - 1) / 2) + (days_left * weeks + days_left * (days_left + 1) / 2);
    }
}

File: Calculate_Money_in_Leetcode_Bank.cpp
class Solution {
public:
    int totalMoney(int n)
{
    int count = 0;
    int sum = 0;
    int q = n/7;
    
    
    int start = 1, end = 7;
    while(q!=-1){
        
        for(int i=start; i<=end && count < n; i++){
            count++;
            cout << count << " ";
            sum = sum + i;
            
        }
        
        start++;
        end++;
        q--;
    }

    return sum;
}
};
